ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutorFactory.java

```diff
@@ -131,11 +128,7 @@ static StandaloneExecutor create(
         udfLoader,
         true,
         versionChecker,
-        (ec, sc) -> InjectorChain.of(
-            new DefaultSchemaInjector(
-                new SchemaRegistryTopicSchemaSupplier(sc.getSchemaRegistryClient())),
-            new DefaultTopicInjector(ec)
-        )
+        Injectors.DEFAULT
```

Comment:
I thought we decided not support schema inference for headless mode (cc @big-andy-coates)?

Reply:
No, we decided to continue to do schema inference to allow users that are already doing it to continue to do so. We just don't support evolution. So if the schema changes and the server restarts the behaviour is undefined.